### PR TITLE
Add weekly update 2016-02-15

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2016-02-15.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Node.js News
+Node v0.10.42 (LTS), Node v0.12.10 (LTS), Node v4.3.0 (LTS) and Node v5.6.0 (Stable) are released.
+
+### February 2016 Security Release Summary
+We had released Node.js [v0.10.42 (Maintenance)](/en/blog/release/v0.10.42/), [v0.12.10 (LTS)](/en/blog/release/v0.12.10/), [v4.3.0 "Argon" (LTS)](/en/blog/release/v4.3.0/) and [v5.6.0 (Stable)](/en/blog/release/v5.6.0/) with fixes for the announced vulnerabilities and updates to OpenSSL.
+
+**Please note that our LTS "Argon" release line has moved from v4.2.x to v4.3.x due to the security fixes enclosed. There will be no further updates to v4.2.x.** Users are advised to upgrade to v4.3.0 as soon as possible.
+
+For the purpose of understanding the impact that the fixed vulnerabilities have on your Node.js deployment and the urgency of the upgrades for your circumstances we are providing details below.
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-02-15.md]: Can these lines be wrapped at 80 chars?

File: locale/en/blog/weekly-updates/weekly-update.2016-02-15.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+See https://nodejs.org/en/blog/vulnerability/february-2016-security-releases/ for more information.
+
+### OpenSSL upgrade summary
+
+Node.js v0.10.42 and v0.12.10 upgrades the bundled version of OpenSSL from 1.0.1q to 1.0.1r. Full details can be found in the [OpenSSL 1.0.1 changelog](https://www.openssl.org/news/cl101.txt).
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-02-15.md]: _Node.js v0.10.42 and v0.12.10 upgrades_ - Shouldn't that be _upgrade_?

**Comment by wonderdogone** [comment]
if @thefourtheye that is indeed correct then the next line below that should also be changed no?
"Node.js v4.3.0 and v5.6.0 upgrades"

**Comment by JungMinu** [comment]
@thefourtheye @wonderdogone Thanks, updated :)

**Pull request merged**